roles/etcd/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Install etcd
+  yum: pkg=etcd state=installed disable_gpg_check=yes
+
+- name: Write etcd global config file
+  template: src=etcd.conf.j2 dest=/etc/etcd/etcd.conf
```

Comment:
@jwhonce This should probably reference the best practices guide and not the style guide. However, does that even apply here? I was under the assumption that the "equals" notation was an ansible DSL and not json. I thought we were just wanting to avoid the use of json style parameters.